yb-voyager/cmd/exportData.go

```diff
@@ -304,9 +305,16 @@ func checkDataDirs() {
 		os.Remove(flagFilePath)
 		os.Remove(dfdFilePath)
 	} else {
-		if !utils.IsDirectoryEmpty(exportDataDir) {
-			//utils.ErrExit("%s/data directory is not empty, use --start-clean flag to clean the directories and start", exportDir)
-			utils.PrintAndLog("%s/data directory is not empty, use --start-clean flag to clean the directories and start", exportDir)
+		status, err := dbzm.ReadExportStatus(filepath.Join(exportDir, "data", "export_status.json"))
+		if err != nil {
+			utils.ErrExit("failed to read export status: %v", err)
+		}
```

Comment:
In the live_migration.go define a function named `isLiveMigrationInSnapshotMode()`. That will make this part of the code a bit more readable. Also consider rearranging the conditions:
```
if !utils.IsDirectoryEmpty(exportDataDir) {
  if !liveMigration || isLiveMigrationInSnapshotMode(exportDir) {
    utils.ErrExit(...)
  }
  utils.PrintAndLog(...)
}
```

Reply:
done